src/components/TimeseriesPreview/TimeseriesPreview.tsx

```diff
@@ -281,9 +317,16 @@ const TimeseriesPreview: React.FC<TimeseriesPreviewProps> = ({
   );
 };
 
+TimeseriesPreview.defaultProps = {
```

Comment:
I agree that such solution works ok in our case, but it looks a bit wrong to have a static prop on function. I would move values in `defaultProps` object directly to props, like:
```typescript
const TimeseriesPreview: FC<TimeseriesPreviewProps> = ({
  color: defaultProps.color
}) => {...}
```
replace with
```typescript
const TimeseriesPreview: FC<TimeseriesPreviewProps> = ({
  color: '#6c65ee'
}) => {...}
```
It's more functional way.

Also, `defaultProps` for FC will be deprecated in the future:
https://twitter.com/dan_abramov/status/1133878326358171650